app/services/hyrax/workflow/workflow_factory.rb

```diff
@@ -2,7 +2,7 @@ module Hyrax
   module Workflow
     class WorkflowFactory
       class_attribute :workflow_strategy
-      self.workflow_strategy = DefaultWorkflowStrategy
+      self.workflow_strategy = Hyrax::Workflow::WorkflowByAdminSetStrategy
```

Comment:
I think it's probably useful to provide, especially for CC migrators who may have gotten to CC 2.0 and not customized their workflow strategy. We had already customized it to support geo concerns.